scripts/globals/mobskills/Slaverous_Gale.lua

```diff
@@ -0,0 +1,28 @@
+---------------------------------------------
+--  Slaverous Gale
+--
+--  Description: Deals earth damage that inflicts Plague and Slow effects on targets in front of the caster
+--  Type: Magical (Earth)
+--
+--
+---------------------------------------------
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+---------------------------------------------
+function onMobSkillCheck(target,mob,skill)
+    return 0;
+end;
+
+function onMobWeaponSkill(target,mob,skill)
+	local typeEffectOne = EFFECT_PLAGUE;
+	local typeEffectTwo = EFFECT_SLOW;
+	MobStatusEffectMove(mob,target,typeEffectOne,1,3,60);
+	MobStatusEffectMove(mob,target,typeEffectTwo,1,3,60);
+	local dmgmod = 1; 
+    local info = MobMagicalMove(mob, target, skill, mob:getWeaponDmg() * math.random(4,6),ELE_EARTH,dmgmod,TP_NO_EFFECT);
```

Comment:
>My indents look ok on Notepad++, I'm not sure what happened there. 

tabs happened, github renders them as inconsistent sizes with no indication that we are looking at a hard tab instead of spaces. In notepad++has a setting that can visual show you your tabs and spaces: `view`\>`show symbol`\>`show white space and tab`